client/src/components/ProductDetails/ImageGallery.jsx

```diff
@@ -1,30 +1,46 @@
-import React, { useState } from "react";
+/* eslint-disable react/prop-types */
+import React, { useState } from 'react';
 import './prodStyles.css';
 
-function ImageGallery ({imageGallery}) {
+function ImageGallery ({styles, currentProductID, currentStyleID}) {
 
-  const [currentImage, updateMainImage] = useState(imageGallery[0])
+  const getCurrentProductObject = (targetId, allProducts) => {
```

Comment:
It looks like the StyleSelector component has a fn with the same name and similar/the same functionality. Maybe it would be helpful to put utility functions in a separate file or you could consider putting constants up at the top (right below your imports)

Reply:
This entire component needs a refactor. For now, the code is fragile but working, so for the purposes of our demo on monday  I opted to leave the code as is. And focus my time on styling Sat and Sunday 

I plan to move all state logic for all of my components over to the ProductOverview container.